packages/core/src/components/Pagination/index.tsx

```diff
@@ -164,16 +177,47 @@ export class Pagination extends React.Component<Props & WithStylesProps> {
             )}
           </Text>
         </div>
-      </Row>
+        <div {...css(styles.next)}>
+          {nextPage}
+          {lastPage}
+        </div>
+      </div>
     );
   }
 }
 
 export default withStyles(
-  () => ({
-    centered: {
-      width: '100%',
-      textAlign: 'center',
+  ({ unit }) => ({
+    wrapper: {
+      display: 'grid',
+      gridTemplateAreas: '"previous page next"',
+      gridTemplateColumns: 'auto 1fr auto',
+      gridColumnGap: unit * 2,
+      alignItems: 'center',
+      justifyItems: 'center',
+    },
+    page: {
+      gridArea: 'page',
+    },
+    previous: {
+      gridArea: 'previous',
+    },
+    next: {
+      gridArea: 'next',
+    },
+    end_align: {
```

Comment:
we do the underscore when it is a modified style, like `button` and `button_red`. here, the alignment styles are modifiers of the `wrapper`, so  `wrapper_centerAlign`, `wrapper_endAlign`, `wrapper_startAlign`. modifier styles usually follow the block they're a modifier of (so after `wrapper`'s styles)